common/views/components/styled/CssGridContainer.tsx

```diff
@@ -98,9 +98,9 @@ const CssGridContainer = styled.div`
       .map(entry => {
         const value = entry[1];
         const { respond, padding } = value;
-        const columns = `[full-start] minmax(${padding}px, 1fr) [main-start] minmax(0, ${themeValues
-          .sizes.xlarge -
-          padding * 2}px) [main-end] minmax(${padding}px, 1fr) [full-end]`;
+        const columns = `[full-start] minmax(${padding}px, 1fr) [main-start] minmax(0, ${
+          themeValues.sizes.xlarge - padding * 2
+        }px) [main-end] minmax(${padding}px, 1fr) [full-end]`;
```

Comment:
Although this is properly linted now, it's not very easy to read. How about:

```suggestion
        const columns = [
          `[full-start] minmax(${padding}px, 1fr)`,
          `[main-start] minmax(0, ${themeValues.sizes.xlarge - padding * 2}px)`,
          `[main-end] minmax(${padding}px, 1fr)`,
          `[full-end]`,
        ].join(' ');
```